src/Andoromeda.Kyubey.Portal/Controllers/AnalysisController.cs

```diff
@@ -5,11 +5,16 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
+using Pomelo.AspNetCore.Localization;
 
 namespace Andoromeda.Kyubey.Portal.Controllers
 {
     public class AnalysisController : BaseController
     {
+        public AnalysisController(ICultureProvider cultureProvider) : base(cultureProvider)
```

Comment:
Please inject ICultureProvider in BaseController

Reply:
Don't use constructor to inject any services

Reply:
done